herbert_core/utilities/geometry/pack_blocks.m

```diff
@@ -0,0 +1,64 @@
+function [positions,free_spaces,last_gap_pos] = pack_blocks(free_spaces,block_sizes,last_gap_pos)
+%PACK_BLOCKS The function places blocks within the array of free spaces
+%trying to leave minimal free space.
+%
+% Inputs:
+% free_spaces  -- 2xn_element array of numbers, defining positions (first
+%                 row and sizes (second row) of gaps to place blocks in.
```

Comment:
```suggestion
%                 row) and sizes (second row) of gaps to place blocks in.
```
sorry to have to correct but it's very difficult to read without the closing bracket